packages/components/src/internal/components/samples/SamplesAssayButton.tsx

```diff
@@ -29,9 +31,15 @@ export const SamplesAssayButton: FC<Props> = memo(props => {
         />
     );
 
+    if (!isLoading(assayModel?.definitionsLoadingState) && assayModel.definitions.length === 0) {
+        items = <MenuItem disabled>No assays defined</MenuItem>;
```

Comment:
I notice that even though disabled the cursor is still the pointer. Not a big deal, of course, but since we don't have a tool tip it looks a little out of place.

Reply:
Yeah, not ideal. But with this being an edge case I am planning to leave it as is for now. Let me know if you disagree and would like me to update.